UnityProject/Assets/Scripts/Core/Chat/ChatRelay.cs

```diff
@@ -205,6 +215,43 @@ bool DistanceCheck(Vector3 playerPos)
 		}
 	}
 
+	private void CheckForRadios(ChatEvent chatEvent)
```

Comment:
this could become expensive, I think

Reply:
if inventories can have all these foreach loops and can work just fine i think this will be fine as well especially with the .5 cooldown